auth.go

```diff
@@ -512,32 +540,28 @@ func (x *Central) RenameIdentity(oldIdent, newIdent string) error {
 		return nil
 	}
 
-	if err := x.authenticator.RenameIdentity(oldIdent, newIdent); err != nil {
-		x.Log.Infof("RenameIdentity failed (%v -> %v) (%v)", oldIdent, newIdent, err)
-		return err
+	var userId UserId
+	var eGetUserId error
+	if userId, eGetUserId = x.userStore.GetUserIdFromIdentity(oldIdent); eGetUserId != nil {
+		x.Log.Infof("Could not find userId for identity failed (%v) (%v)", oldIdent, ErrIdentityAuthNotFound)
```

Comment:
Typo - extra word "failed". Also, same comment here as above, about idomatic Go code and `if` statements.